src/modules/network/components/DialogAddNetwork/DialogAddNetwork.js

```diff
@@ -119,10 +132,37 @@ const DialogAddNetwork = () => {
                 pattern: { value: regex.url, message: t('Invalid URL') },
               })}
             />
-            <PrimaryButton type="submit" className={`${styles.button}`}>
+            {!isNetworkUrlOk && !isAddingNetwork && (
+              <span className={styles.connectionFailed}>
+                <span className={styles.errorText}>
+                  {t('Could not fetch chain ID. Is your RPC URL correct?')}
```

Comment:
rest all good, can you use isOnchainOK and isOffchainOK to create a message?

e.g `Failed to fetch !isOnchainOK && 'onchain', !isOffchainOK && 'offchain'. Please check the URL'

Reply:
https://github.com/LiskHQ/lisk-desktop/pull/5239/commits/665cd86be44b0a0963e784bbfd75499004f0794b